Test/comp/DowncastDt.dfy

```diff
@@ -8,22 +8,41 @@
 // Java avoids this problem by not even allowing traits in datatypes.
 
 datatype Co<+T> = Co(T)
+datatype Con<-T> = Con(T -> bool)
```

Comment:
Can you rename Con to Cont? If we ever compile this in Java in Windows, it will trigger problems.

Reply:
Now the CI notices that for us. Thanks!